### PR TITLE
schemas, tests, and code for urls -fixes #483

### LINKED ISSUE #483: issue and website URLs should go in the `described.urls` property
We semi-recently added a `described.urls` property to point to the download location etc for a component. We previously had `issueTracker` and `projectWebsite` properties. These latter two should be moved into the `urls` property. 

Note that the website may use these so may also need updating.

### PR DESCRIPTION
**Pull request by ignacionr**
The change surface looks big, but it mainly affects tests and their assets (a good thing).
Modifies schemas as per #483

### CODE REVIEW

File: providers/search/azureSearch.js [hunk 1]
```diff
@@ -412,8 +412,8 @@ class AzureSearch extends AbstractSearch {
         { sourceFieldName: '/described/sourceLocation/url', targetFieldName: 'sourceLocation' },
         { sourceFieldName: '/described/files', targetFieldName: 'fileCount' },
         { sourceFieldName: '/described/releaseDate', targetFieldName: 'releaseDate' },
-        { sourceFieldName: '/described/projectWebsite', targetFieldName: 'projectWebsite' },
-        { sourceFieldName: '/described/issueTracker', targetFieldName: 'issueTracker' },
+        { sourceFieldName: '/described/urls/projectWebsite', targetFieldName: 'projectWebsite' },
+        { sourceFieldName: '/described/urls/issueTracker', targetFieldName: 'issueTracker' },
```
Comment (dabutvin) [inline providers/search/azureSearch.js]: @geneh  make sure we create the new indexer when this change lands

**Comment by dabutvin** [comment]
we need to rev the schema version in definitionService for this change

**Comment by storrisi** [comment]
@ignacionr are we able to move forward this one?

**Comment by nellshamrell** [comment]
I apologize that this was open for so long without more feedback. Due to the age of the pull request, as well as the merge conflicts, I am going to close this for now. However, @ignacionr, if you are willing to repair the merge conflicts please feel free to re-open this and we can look at it again. If you have any questions, please feel free to reach out to me here or on Discord.

**Pull request closed without merging**